src/Kentico.Xperience.Lucene.Admin/Client/src/models/LuceneIndexConfigurationComponentClientProperties.tsx

```diff
@@ -0,0 +1,11 @@
+import { FormComponentProps } from '@kentico/xperience-admin-base';
+import { LuceneIndexContentType } from './LuceneIndexContentType';
+import { LuceneIndexChannelConfiguration } from './LuceneIndexChannelConfiguration';
+import { LuceneIndexChannel } from './LuceneIndexChannel';
+
+export interface LuceneIndexConfigurationComponentClientProperties
```

Comment:
Change name to LuceneIndexConfigurationFormComponentProps to match naming guidelines.